### PR TITLE
Upload Instructions Update

### PR DESCRIPTION
**Pull request by cmcphilamy**
In order to meet the Oct 5th milestone the directions have been updated with naming convention information and better upload instructions.

### CODE REVIEW

**Comment by cmcphilamy** [comment]
Making changes to facilitate Oct 5th Milestone.  Changes include bucket naming convention and file naming conventions.  Also removed language mentioning alpha.

File: data_steward/spec/pages/file_transfer_procedures.md [hunk 1]
```diff
@@ -15,24 +15,40 @@ template: page
...
   
       Keep this information in a safe place and do not share it with others.
 
-## Upload the Test Data Set
+## File Naming Convention
+
+`<Bucket name>_<quarter end date (yyyymmdd)>_<table name>.csv`
 
-__Note: You should not upload real participant data or de-identified data during alpha. You will be given a test data set to use instead.__
+An example is:
```
Comment (mark-velez) [inline data_steward/spec/pages/file_transfer_procedures.md]: Will put back note

File: data_steward/spec/pages/file_transfer_procedures.md [hunk 2]
```diff
@@ -15,24 +15,40 @@ template: page
...
+* AoUSite123_20170930_person.csv
+* AoUSite123_20170930_visit_occurrence.csv
+* AoUSite123_20170930_condition_occurrence.csv
 
-The instructions below assume the bucket assigned to you was `test-site-bucket-aou-data-steward-nyc`. You should adapted according to the information that was sent to you by `sysadmin@pmi-ops.org`:
+## Prepare Files to be Uploaded
+
+For this test the same files and file structure will be used as in the [Data Sprint](https://cumc-dbmi.github.io/pmi_sprint_reporter/data_model.html).
```
Comment (mark-velez) [inline data_steward/spec/pages/file_transfer_procedures.md]: For new HPO sites, we should streamline experience from sprint to real EHR data submissions

File: data_steward/spec/pages/file_transfer_procedures.md [hunk 3]
```diff
@@ -15,24 +15,40 @@ template: page
...
+ * Person
```
Comment (mark-velez) [inline data_steward/spec/pages/file_transfer_procedures.md]: Merge this in, this may need a better place

**Comment by mark-velez** [comment]
Closing. We are now using another approach for the doc site.

**Pull request closed without merging**